lms/djangoapps/courseware/views.py

```diff
@@ -892,7 +895,18 @@ def course_about(request, course_id):
                     shoppingcart.models.CourseRegCodeItem.contained_in_order(cart, course_key)
 
             reg_then_add_to_cart_link = "{reg_url}?course_id={course_id}&enrollment_action=add_to_cart".format(
-                reg_url=reverse('register_user'), course_id=urllib.quote(str(course_id)))
+                reg_url=reverse('register_user'), course_id=urllib.quote(str(course_id))
+            )
+
+        ecommerce_checkout_link = ''
```

Comment:
Please add comments explaining what is going on here. I'm not sure what you're doing or why.